planner/README.md

```diff
@@ -0,0 +1,67 @@
+SeaClouds Planner
+==================
+
+This component provides design time functionalities for the SeaClouds Platform. 
+It provides planning, replanning, DAM generation and monitoring rules retrieval via rest API.
+
+# Installing and running SeaClouds Planner
+The easiest way to install and run this component is explained in SeaClouds [README.md](../README.md).
+
+## Requirements
+Running SeaClouds Planner only requires Java 7 (or greater) installed on the target machine. Among with the software requirements, it also needs:
+
+- SeaClouds Discoverer, Sla, Monitor components up and running on an accessible endpoint.

```

Comment:
There is no bootstrap order, if the all can be deployed in parallel is ok. If an order is requested any permutation of such service is suitable.